cli/cmd/help.go

```diff
@@ -171,6 +180,50 @@ Usage: {{ .Cmd }}
   {{.Example}}
 {{end -}}
 
+{{- if .HasAvailableSubCommands}}
+Use "{{.CommandPath}} <command> --help" for more information about a command.
+{{end -}}
+`
+
+	usageTemplateWithoutExternal = util.Bold("USAGE") + `
```

Comment:
As I can see, these usage templates are almost the same. The only difference is external commands descriptions. Please, remove duplication.

Reply:
Removed